### ISSUE TITLE
Remove "Player Enter"/"Player Exit" Popups

### ISSUE THREAD

**Issue by TheMan6900** [EmeraldLoc/tag#678]
They serve no purpose, and just take up screen space

**Comment by EmeraldLoc** [comment]
Not possible right now, or else I would have.

**Comment by TheMan6900** [comment]
Werid how that isnt possible to do,

**Comment by Bear64DX** [comment]
Wait doesn't Geoguessr do this though? I think it's possible

**Comment by EmeraldLoc** [comment]
Let me rephrase it. It IS possible, just breaks all mod support, and breaks the disabled popup setting.

**Comment by Bear64DX** [comment]
Ohhh

**Comment by TheMan6900** [comment]
Pretty sure this can be properly added now

**Comment by EmeraldLoc** [comment]
How would that be done?

**Comment by TheMan6900** [comment]
Pretty sure something got added that allows for it (my issue on it got accepted and completed so)

**Comment by TheMan6900** [comment]
This actually can be added now 👍

**Comment by Bear64DX** [comment]
I thought `HOOK_ALLOW_WARP_POPUP` wasn't added into dx 1.1
